pele/potentials/_pele.pyx

```diff
@@ -59,3 +59,20 @@ cdef class BasePotential(object):
                                              array_wrap_np(hess),
                                              eps)
         return np.reshape(hess, [x.size, x.size])
+
+cdef class SimplePairwisePotential(BasePotential):
+    """
+    this class has additional methods that are common for simple pairwise potentials
+    """
+    def evaluatePairPotential(self, rmin, rmax, nrpoints=25, atomi=0, atomj=0):
+        """
+        this function is a utility to compute the energy as a function of interatomic distance
+        between two atoms (atomi and atomj)
+        """
+        cdef np.ndarray[double, ndim=1] x = np.zeros(nrpoints)
+        cdef np.ndarray[double, ndim=1] y = np.zeros(nrpoints)
+        self.spp_ptr.get().evaluate_pair_potential(rmin, rmax, atomi, atomj,
+                                                   array_wrap_np(x), array_wrap_np(y))
+        return x, y
+
+
```

Comment:
ok, I have undone all the mess. Turns out that you can revert single files by using this really neat command
git show <commit> -- <path> | git apply -R
so it really took 2 minutes to untangle all the unwanted changes. I'll see that I reimplement the plotting feature in python. The reason why it was written in cpp was that one would call directly the interaction, but I agree that it can be done also with simple get_energy calls.